q2cli/tools.py

```diff
@@ -134,11 +138,16 @@ def view(visualization_path, index_extension):
                     break
 
 
-@tools.command(help='Extract a QIIME Artifact or Visualization.')
+@tools.command(short_help="Extract a QIIME Artifact or Visualization archive.",
+               help="Extract all contents of a QIIME Artifact or "
+                    "Visualization's archive. Use 'qiime tools export' to "
```

Comment:
How about we clarify that there are extra "things" involved in an archive, beside just the source data, that way we can further differentiate from `export`. Maybe something like: `... Visualization's archive, including provenance, metadata, and actual data. Use ...`?